### DISCUSSION TITLE
Rustdesk Pro MTA relay

### DISCUSSION THREAD

**Discussion by HGrundeman** [rustdesk/rustdesk-server-pro#297] [Q&A]
Hi, 

I want to send e-mails from RD Pro server via our MTA to some domain mailadres. I've filled the SMTP-host and port fields and user-authentication but It seems that these SMTP-host settings aren't used at all. The system tries to relay via the MX-record of the from: mailadres. E.g. mailing with the from-address rustdeks@vm1cloud.nl results in the server trying to deliver via the mx-host of vm1cloud.nl server instead of relaying via the MTA. Same if I put rustdesk@vm1.nl in the from-field.

I cannot find documentation about the e-mail. Who can help me with this?

Thanks in advance.
Hans.

**Comment by fufesou** [comment]
Hi, we do not support sending emails via the MTA at this time.

**Reply by HGrundeman** [reply]
OK, then it makes sense it does not work. What is the purpose of the SMTP-settings?

Regards Hans.

**Reply by fufesou** [reply]
Sorry, I just didn't know much about sending emails via the MTA before. So I didn't implement it.

The SMTP settings are mainly used for sending messages, such as login verification, warning information, etc.
